### ISSUE TITLE
Supporting Stable Liightning 4.0 (NOT GREATER)  and EWS Beta 3.4 beta3

### ISSUE THREAD

**Issue by bavincen** [Ericsson/exchangecalendar#247]
We are presently planning to support lightning 4.0 stable .  If you are using lightning version any greater than 4.0 we are not supporting that until we release stable EWS 3.4 

Reasons are
We have/had quite big number of release items which are not comparable with 4.0 to be fixed.
Many issues raised in existing code after strict mode enabled in Thunderbird/lightning in their latest versions

so please downgrade to lightning 4.0 stable and contribute us 

Thanks
For your comment and views

**Comment by dicer** [comment]
Does that include 4.0.0.1? I just upgraded my Thunderbird to 38.0.1, got auto-upgraded to Lightning 4.0.0.1 and can't downgrade to 3.3.3 as it is not compatible.
I am running EWS Provider 3.4.0~beta3
Now my exchange calendar doesn't work anymore. I deleted it and can't add it anymore. I have to enter my password multiple times and in the end always get:

Error during checking of server and mailbox: HTTP Client error 401: Unauthorized URL: https://..../EWS/Exchange.asmx   (Code: -19)

The URL in the browser forwards me to /EWS/Services.wsdl.
